src/acceptance/assets/app/go_app/deploy.sh

```diff
@@ -1,37 +1,44 @@
 #!/bin/bash
-
 set -euo pipefail
-config=${CONFIG:-}
+
+config="$(cat "${CONFIG:-}")"
+
 function getConfItem(){
-  val=$(jq -r ".$1" "${config}")
+  val=$(jq -r ".$1" <<< "${config}")
```

Comment:
This makes `getConfItem` working independently from the current directory

Reply:
I think a comment here may be helpful. When reading it is not obvious to me why we switch here from a single call to one making use of stdin. The content what `jq` gets via stdin (in the rewritten case) or in the invocation (in the original case) is probably identical.